src/common-components/ThirdPartyAuthAlert.jsx

```diff
@@ -20,7 +21,10 @@ const ThirdPartyAuthAlert = (props) => {
     message = formatMessage(messages['register.third.party.auth.account.not.linked'], { currentProvider, platformName });
   }
 
-  if (!currentProvider) {
+  if (currentProvider) {
+    // Setting this cookie to capture marketingEmailsOptIn for SSO flow on the onboarding component
+    setCookie('ssoPipelineRedirectionDone', true);
+  } else {
```

Comment:
i am confused. wasn't this only part of 2u-main branch? why is it appearing as new change?

Reply:
Last time we rebased the branch and resolved the commit message linting issue by editing and removing commit messages that disturbed the commit history I think that could be the reason.